packages/wallets/src/components/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -1,20 +1,58 @@
 import React from 'react';
+import classNames from 'classnames';
+import { PrimaryActionButton } from '../PrimaryActionButton';
 import { WalletModal } from '../WalletModal';
 import { WideWrapper } from '../WideWrapper';
+import JurisdictionCard from './JurisdictionCard';
 import './JurisdictionModal.scss';
 
 const JurisdictionModal = () => {
+    const [selectedJurisdiction, setSelectedJurisdiction] = React.useState('');
+
+    const jurisdictions = ['St. Vincent & Grenadines', 'British Virgin Islands', 'Vanuatu'];
+
     return (
         <WalletModal>
             <WideWrapper
                 renderFooter={() => (
                     <React.Fragment>
-                        <button className='wallets-jurisdiction-modal__button'>Next</button>
+                        <PrimaryActionButton
+                            className={classNames('wallets-jurisdiction-modal__button', {
+                                'wallets-jurisdiction-modal__button--disabled': !selectedJurisdiction,
+                            })}
+                        >
+                            Next
+                        </PrimaryActionButton>
                     </React.Fragment>
                 )}
                 renderHeader={() => <div>Choose a jurisdiction for your MT5 Derived account</div>}
             >
-                <div className='wallets-jurisdiction-modal'>BODY</div>
+                <div className='wallets-jurisdiction-modal'>
+                    <div className='wallets-jurisdiction-modal__cards'>
+                        {jurisdictions.map((jurisdiction, i) => (
+                            <JurisdictionCard
+                                isSelected={selectedJurisdiction === jurisdiction}
+                                jurisdiction={jurisdiction}
+                                key={i}
+                                onSelect={clickedJurisdiction => {
+                                    setSelectedJurisdiction(clickedJurisdiction);
+                                }}
+                                tag='Straight-through processing'
+                            />
+                        ))}
+                    </div>
+
+                    {selectedJurisdiction && (
+                        <div className='wallets-jurisdiction-modal__tnc'>
+                            Add Your Deriv MT5 Financial account under Deriv (V) Ltd, regulated by the Vanuatu Financial
```

Comment:
Its just for Financial Labuan right?
What about other jurisdiction?

Reply:
Didnt want to pack this PR. If we do logic it will be 50 file changes.

Reply:
Yeah sound Good to Me
Approved